reloader/overlays/moc/smaug/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
What namespace should this app be deployed? 

Depending that, we might need to create the namespace. We will also want to create an ArgoCD app to deploy everything in `reloader/overlays/moc/smaug`. Can you add that too? 

The docs for that are here: 
https://www.operate-first.cloud/apps/content/argocd-gitops/add_application.html

Reply:
@HumairAK In my Quicklab cluster I had the reloader deployment in its own namespace called reloader per the kustomize install instructions. So I believe yes we will need to create a new namespace. And I'll add the namespace field to the kustomization.yaml 
Also working on the ArgoCD app.

Reply:
@dystewart okay great, just fyi we manage namespaces in the `cluster-scope` directory, as it's a perivileged resource
so to create this namespace, create it here: 

https://github.com/operate-first/apps/tree/master/cluster-scope/base/core/namespaces 

follow the examples of the others, pick a suitable quota, if a custom one is required then add one as needed (you will need to know what resources the operator requires to runs, I imagine it's quite minimal, but you can check your quicklab deployment) 

https://www.operate-first.cloud/apps/content/cluster-scope/add_resource_quotas.html